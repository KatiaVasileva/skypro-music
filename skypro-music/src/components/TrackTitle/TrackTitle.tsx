import Icon from "../Icon/Icon";
import styles from "./TrackTitle.module.css";
import classNames from "classnames";

function TrackTitle() {
  return (
    <div className={styles.contentTitle}>
      <div
        className={classNames(styles.playlistColumn, styles.playlistColumn01)}
      >
        Трек
      </div>
      <div
        className={classNames(styles.playlistColumn, styles.playlistColumn02)}
      >
        Исполнитель
      </div>
      <div
        className={classNames(styles.playlistColumn, styles.playlistColumn03)}
      >
        Альбом
      </div>
      <div
        className={classNames(styles.playlistColumn, styles.playlistColumn04)}
      >
        <Icon iconClass={styles.playlistTitleSvg} name="icon-watch" />
      </div>
    </div>
  );
}

export default TrackTitle;
