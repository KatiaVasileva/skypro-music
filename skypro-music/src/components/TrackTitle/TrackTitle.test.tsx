import ReduxProvider from "@/store/ReduxProvider";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import TrackTitle from "./TrackTitle";

describe("Search component", () => {
  it("renders correctly", () => {
    const { container } = render(
      <ReduxProvider>
        <TrackTitle />
      </ReduxProvider>
    );
    expect(container).toMatchSnapshot();
    expect(screen.getByText("Трек")).toBeInTheDocument();
    expect(screen.getByText("Исполнитель")).toBeInTheDocument();
    expect(screen.getByText("Альбом")).toBeInTheDocument();
  });
});
