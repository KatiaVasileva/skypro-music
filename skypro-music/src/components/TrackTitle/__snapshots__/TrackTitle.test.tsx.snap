// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders correctly 1`] = `
<div>
  <div
    class="contentTitle"
  >
    <div
      class="playlistColumn playlistColumn01"
    >
      Трек
    </div>
    <div
      class="playlistColumn playlistColumn02"
    >
      Исполнитель
    </div>
    <div
      class="playlistColumn playlistColumn03"
    >
      Альбом
    </div>
    <div
      class="playlistColumn playlistColumn04"
    >
      <div
        class=""
      >
        <svg
          class="playlistTitleSvg"
        >
          <use
            xlink:href="/img/icon/sprite.svg#icon-watch"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
